src/main/java/commands/AddMangaCommand.java

```diff
@@ -36,16 +40,23 @@ public void execute(Ui ui, AuthorList authorList) throws TantouException {
                 existingAuthor.addManga(incomingManga);
                 System.out.printf("Manga %s added successfully to author %s\n",
                         incomingManga.getMangaName(), existingAuthor.getAuthorName());
+
+                // Assert that the manga was successfully added
+                assert authorList.getAuthor(incomingAuthor).hasManga(incomingManga) : "Failed to add manga";
+
                 return;
             }
 
             // Exception is thrown when adding manga that already exists with author
+            assert authorList.getAuthor(incomingAuthor).hasManga(incomingManga) : "Manga does not actually exist!";
```

Comment:
Perhaps a bit more clarity in the message could be useful here, like "Detected that the manga exists, but manga does not actually exist."